weblab/repocache/migrations/0015_auto_20191128_1331.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-11-28 13:31
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repocache', '0014_auto_20191121_1448'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='cachedmodelversion',
+            name='message',
+            field=models.TextField(default=' ', help_text='Git commit message'),
+        ),
+        migrations.AddField(
+            model_name='cachedprotocolversion',
+            name='message',
+            field=models.TextField(default=' ', help_text='Git commit message'),
+        ),
+        migrations.AlterField(
+            model_name='cachedmodelversion',
```

Comment:
could you help me understand why there is an AlterField migration done to timestamp please

Reply:
is it because of the addition of help text?

Reply:
I've convinced myself that its because of the help text so obvious question - sorry